tmk_core/common/action.c

```diff
@@ -562,8 +562,22 @@ void process_action(keyrecord_t *record, action_t action) {
                     /* tap key */
                     if (event.pressed) {
                         if (tap_count > 0) {
-                            dprint("KEYMAP_TAP_KEY: Tap: register_code\n");
-                            register_code(action.layer_tap.code);
+#        ifdef IGNORE_LAYER_TAP_INTERRUPT
+                            if (
+#            ifdef IGNORE_MOD_TAP_INTERRUPT_PER_KEY
+                                !get_ignore_mod_tap_interrupt(get_event_keycode(record->event, false), record) &&
```

Comment:
Not a fan of this using the same function here.  Would prefer a different function or a generic name for both.